### PR TITLE
Edits from docs team

### PR DESCRIPTION
**Pull request by madeline-brennan**
Made a few tweaks

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1322,24 +1322,24 @@ This request retrieves a list of all users in the order that they were added.
 **Sorting**
 
 If you use the `sort` query parameter, you can only sort the list of users by 1 field. 
-You can add a `+` or `-` to display the list in ascending or descending order respectively.
+To display the list in ascending or descending order respectively, you can add a `+` or `-` y.
```
Comment (george-haddad) [inline apiary.apib:1325]: There is a trailing `y` character a the end here, looks like left over from the word `respectively`

Comment (george-haddad) [reply apiary.apib:1325]:
```suggestion
To display the list in ascending or descending order respectively, you can add a `+` or `-`.
```

**Review by george-haddad — APPROVED** [review]
:+1:

File: apiary.apib [hunk 2]
```diff
@@ -1322,24 +1322,24 @@ This request retrieves a list of all users in the order that they were added.
...
+- The query parameter `licenseType` sorts by two fields: `licenseType` and `licenseStatus`.
+- To sort by a custom field, the sort value needs to include `customFields`, `.`, and the `{key}`. The key is the name of the custom field.
 
-So that the behavior of sorting custom fields is consistent, the following special characters can safely be sorted: `` !@$^*()_+-=,./;'[]<>?:|~` ``
-Other special characters may cause unexpected behavior when sorting.
+To make sure that the sorting behavior for custom fields is consistent, the following special characters can be sorted: `` !@$^*()_+-=,./;'[]<>?:|~` ``
+If you use any additional special characters, the sorting behavior may be inconsistent.
 
-So that the `+` character encoded to `%2B` stays valid, make sure that the query parameters are encoded in accordance
+So that the `+` character encoded to `%2B` stays valid, the query parameters need to be encoded in accordance
```
Comment (tkrisztian95) [inline apiary.apib:1340]:
```suggestion
So that the `+` character encoded to `%2B` stays valid, the query parameters need to be encoded in accordance.
```

Comment (madeline-brennan) [reply apiary.apib:1340]: This continues on the next line, so period not necessary here

Comment (madeline-brennan) [reply apiary.apib:1340]: In this case, the period continues on the next line

**Pull request merged**